.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,24 @@
+# Release Description
+<!--- Summarize what is being released.  -->
+
+# Release Checklist
+<!--- Fill out the following Release checklist -->
+- [ ] Version number has been updated
+- [ ] Description file updated with New Developers (if applicable)
+- [ ] NEWS.md has been updated and issues linked where appropriate
+- [ ] Ensure all unit tests are passing and Test Coverage is Green on Readme
+- [ ] Ensure CMD Checks are passing and says Passing on Readme
+- [ ] GitHub actions on this PR are all passing
+- [ ] Draft GitHub release created using automatic template and updated with additional details. Remember to click "release" after PR is merged.
+- [ ] Website is deployed and visual checks for integrity completed
+- [ ] Prepare and Post announcements for Linked-In and Slack Channels for Release
+
+# Clean up Checklist
+<!--- Fill out the following Clean up checklist -->
+- [ ] Delete branches that are stale or abandoned 
+- [ ] Delete branches from merged Pull Requests that made it into `devel`
```

Comment:
Do we need this one? The feature branch is deleted automatically if it is merged.

Reply:
I've noticed a few that get left behind.